### DISCUSSION TITLE
3004.388.8_4-gnuton0_alpha1

### DISCUSSION THREAD

**Discussion by gnuton** [gnuton/asuswrt-merlin.ng#710] [Announcements]
##  GNUton's Asus Merlin changelog  ##

Date: November 24, 2024
Release: 3004.388.8_4-gnuton1

  - CHANGED: VPN killswitch will now only be active if the
             VPN client itself is enabled.  If you stop/start
             the client yourself over SSH, you need to also
             update the enabled/disabled nvram setting.
  - FIXED: Security issues in AiCloud (backports from Asus)
  - FIXED: CVE-2024-2511, CVE-2024-4741, CVE-2024-5535 &
           Implicit rejection for RSA PKCS#1 in openssl
           (backport from Ubuntu by RSDNTWK)

<hr /><em>This discussion was created from the release <a href='https://github.com/gnuton/asuswrt-merlin.ng/releases/tag/3004.388.8_4-gnuton0_alpha1'>3004.388.8_4-gnuton0_alpha1</a>.</em>

**Comment by alfky12** [comment]
@gnuton dirty flash for tuf-ax3000(v1) since 13 hour ago and still going strong. But UI related issue like #676 and #685 still not fixed. For VPN, I just use openvpn server for local connections from outside network, and wireguard client for DNS over WARP, and it seems fine.

**Reply by gnuton** [reply]
Yep I will check those bugs ASAP.

**Comment by ZliTebra** [comment]
Hi,Is there any possibility to improve Gaming for Tuf ax5400? Any new improved features for gaming?  Thank you!

**Reply by gnuton** [reply]
Hi,
this release will have mostly bugfixes and security improvements.
No new features. What's the matter with gaming features?

**Reply by ZliTebra** [reply]
The model Tuf Ax5400 is a bit older router, but it is still really good.  Thank you also for every new Firmware and for your hard work to improve the whole system even better.  Gaming is decent, but I wonder if there is a possibility in the next firmware to be an option for Tuf AX5400 FlexQOS, or something similar where some additional settings could be made for a better online gaming experience?  Thank you very much!

**Reply by gnuton** [reply]
FlexQOS can be supported if not there. Can you check if there in the QOS page?

**Reply by ZliTebra** [reply]
There is none on the router.  Cake was not an option either, and then it appeared with one of the new firmwares. That's why I was thinking if it is possible to upgrade?

**Reply by gnuton** [reply]
Please open a ticket for this and let's continue the discussion there. I will double check if that's possible.

**Reply by BoostOver** [reply]
> Please open a ticket for this and let's continue the discussion there. I will double check if that's possible.

TUF AX5400 has QoS adaptive.
FlexQoS is a Addon for QoS adaptive.
Via AMTM istall FlexQos
Simple😉

Se ho scritto male….traduci per me.
Grazie

**Reply by ZliTebra** [reply]
I don't understand.  Are you saying that you have the FlexQos option on the Tuf Ax5400?
Do you have a link where I could watch it, download it?  I don't understand.Tnx

**Comment by ZliTebra** [comment]
The model Tuf Ax5400 is a bit older router, but it is still really good.
Thank you also for every new Firmware and for your hard work to improve the
whole system even better.  Gaming is decent, but I wonder if there is a
possibility in the next firmware to be an option for Tuf AX5400 FlexQOS, or
something similar where some additional settings could be made for a better
online gaming experience?  Thank you very much!

**Reply by ZliTebra** [reply]
Maybe the WTfast option to insert the same for all slightly older routers?  Can something like that be done?

**Comment by ZliTebra** [comment]
I would be grateful if something could be done for gaming in the next firmware.  Thanks again for everything.  All the best!

**Comment by denisgersh** [comment]
The uptime is 21.5 days, and the firmware has been operating without any notable issues on XT8 (Ver. 1). Thanks 👍

Switched from the official ASUS firmware v 3.0.0.4.388_24688 without wiping/resetting after the installation.

**Reply by cw2387** [reply]
This "alpha" FW is rock solid on my two ET8 units (main/remote).  Gnuton FW has proven to be much more stable for me than the stock FW.  Uptime so far: 14 days 23 hour(s) 32 minute(s) 47 seconds

**Comment by Bob575** [comment]
Running with no issues on RT-AX58U_V2 as mesh node since 1 Dec 24.
Excellent Work! Thank You!

**Comment by BoutTime2** [comment]
Any chance of enabling 5.9Ghz for dedicated wireless backhaul on the XT8s? I'm in the UK in case it's region specific. Cheers.

**Comment by jimmiedave** [comment]
Today my RT-AX3000 told me there was a Merlin firmware update, but upon clicking I get no release notes, and a 404 on clicking to download site. Trouble with the build process?

I check weekly for updates (to this and a lot of stuff) but since this Alpha is vintage 11/24, why am I getting the cage rattled in early Jan 2025?

I appreciate all the work that goes into making this firmware available for us. I do not take it for granted. Thanks so much!

**Reply by gnuton** [reply]
the build process was all good I had to verify and approve the releases...I may need to move the notification after the publishing.
